writeengine/server/we_dmlcommandproc.cpp

```diff
@@ -3121,6 +3121,20 @@ uint8_t WE_DMLCommandProc::processUpdate(messageqcpp::ByteStream& bs,
                             break;
                         }
 
+                        case CalpontSystemCatalog::LONGDOUBLE:
+                        {
+                            double dll = row.getDoubleField(fetchColPos);
+
+                            if (dll == std::numeric_limits<double>::infinity())
+                                continue;
+
+                            ostringstream os;
+                            //@Bug 3350 fix the precision.
+                            os << setprecision(16) << dll;
```

Comment:
I see setprecision(16) for the double case. Is this also accurate for long double?

Reply:
I'm not sure. I set it to 19. Hopefully that will be OK. 7 for FLOAT above also seems strange.

Reply:
https://en.cppreference.com/w/cpp/types/numeric_limits/digits10

Precision for float, double, and long double is 6, 15, and 18 respectively. We are setting them to one higher than the standard. I believe this is happening throughout the code base.